### ISSUE TITLE
Bulk ignore fields

### ISSUE THREAD

**Issue by torronen** [dotnet/machinelearning-modelbuilder#2363]
**Is your feature request related to a problem? Please describe.**
Probably a rare case , but my datasets have lots of features I may want to ignore.
Feel free to close if this too much of an rare edge case.

In this case I want to ignore all 1000 fields / columns starting with "Embed"  

![embeddings-ignore](https://user-images.githubusercontent.com/26261427/201219863-745cce15-909d-4257-ab6b-2173e67a4d25.jpg)

**Describe the solution you'd like**
A way to select all columns which match the search criteria and bulk change their settings.

**Describe alternatives you've considered**
Click, I, Enter => This sequence works okay for less than 100 columns, but currently it is too much.

**Comment by beccamc** [comment]
@luisquintanilla This is an interesting idea to explore. I think we've gotten feedback before that a bulk edit would be helpful. I don't know that we want to write a lot of complicated UI, but maybe there is another way for us to provide more support here?

**Comment by luisquintanilla** [comment]
I agree. Interesting idea to explore for large datasets. Maybe something similar to the "Categorical" checkbox. We could have an "Ignore" checkbox and remove it from the "Column Purpose" dropdown. You could "Check all" which would ignore everything and then you can uncheck only the ones you actually want to use.

**Comment by torronen** [comment]
@luisquintanilla Sound good! Even better if it would be possible to combine search/filter on top right corenet and then "check all" would only affect the filtered items.
